assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -15519,4 +15531,54 @@ public void setSubmissionTimestamp(String timeStamp) {
             m_timeStamp = timeStamp;
         }
     }
+
+    /**
+     * Used when students are uploading a video object
+     */
+    private List<Reference> getVideoAttachments(RunData data, SessionState state, Assignment assignment) {
+        List<Reference> attachments = entityManager.newReferenceList();
+        //Get the video object in base64 from the form post
+        String videoResponseBase64 = data.getParameters().get("videoResponse");
+        if(StringUtils.isBlank(videoResponseBase64)){
+            return attachments;
+        }
+        videoResponseBase64 = videoResponseBase64.split(",")[1];
```

Comment:
this could blow up. It needs some checking before accessing it.

Reply:
That makes sense, I'll include a length check in the if.